server/src/main/java/com/google/android/attestation/AuthorizationList.java

```diff
@@ -777,307 +783,117 @@ private static void addOptionalAttestationApplicationId(
   }
 
   public static Builder builder() {
-    return new Builder();
+    return new AutoValue_AuthorizationList.Builder()
+        .setPurpose(ImmutableSet.of())
```

Comment:
Why do the booleans need to be initialized with a default value?

Reply:
go/autobuilder#required-optional-and-nullable-parameters

`IllegalStateException` is thrown unless a default value is provided.